src/ecotranslations.jl

```diff
@@ -0,0 +1,13 @@
+# Getters and setters with microbiome-flavored names
+
+# species -> features
+nfeatures(com::AbstractComMatrix) = nspecies(com)
```

Comment:
throughout here, I wonder if something like `const nfeatures = nspecies` might not be better?

Reply:
I'll try it.